logstash-offline/roles/logstash-ansible/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for logstash-ansible
+userName: "{{ username }}"
+project_name: "%{[fields][site]}-%{[fields][technology]}-%{[fields][log]}-%{[fields][foundation]}" #
+hostname: "%{[host][hostname]}"
+component_name: "%{[fields][technology]}"
```

Comment:
I would add documentation here indicating that these variables are using the logstash formatting for referencing the fields from filebeat. 

If they need to update the filebeat field referenced, they can update these variables.